stl/inc/span

```diff
@@ -206,7 +206,10 @@ struct _Span_extent_type {
 
     constexpr _Span_extent_type() noexcept = default;
 
-    constexpr explicit _Span_extent_type(const pointer _Data, size_t) noexcept : _Mydata{_Data} {}
+    constexpr explicit _Span_extent_type(const pointer _Data, size_t) noexcept : _Mydata{_Data} {
+        static_assert(
+            _Mysize <= dynamic_extent / sizeof(_Ty), "size of span in bytes exceeds numeric_limits<size_t>::max()");

```

Comment:
No change requested. Should we file an LWG issue as this affects well-formedness?

Reply:
Yeah, I would appreciate more LWG guidance on what to do here.

Reply:
Filed #5280 for followup, thanks again.